app/freedomvote/settings.py

```diff
@@ -34,6 +34,14 @@
     'PIWIK'                : {
         'SITE_ID'          : 0,
         'URL'              : '',
+    },
+    'EMAIL'                : {
+        'HOST'             : 'localhost',
+        'HOST_USER'        : '',
+        'HOST_PASSWORD'    : '',
+        'PORT'             : '25',
+        'USE_TLS'          : 'False',
+        'USE_SSL'          : 'False',
```

Comment:
Can we add the EMAIL_BACKEND (https://docs.djangoproject.com/en/2.0/topics/email/#email-backends) in here as well? Would be very useful, for example when devleoping etc.

Reply:
... could be optional of course.

Reply:
Done!